pkg/util/github/github.go

```diff
@@ -190,7 +190,16 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 	binaryPath := ""
 	switch {
 	case strings.HasSuffix(assetTmpFile, TAR):
-		tarFile := assetTmpFile[:len(assetTmpFile)-3]
+		tarFile := ""
+		if osType == minishiftos.LINUX {
+			// https://github.com/openshift/origin/issues/12663
```

Comment:
-1, this is just putting a bandage on the problem. It break backwards compatibility, is heard to understand and does not tackle the problem in its core. As said, we should be a bit smarter here and don't rely on the what we think is in the tar ball.